upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.8.yaml.template

```diff
@@ -0,0 +1,285 @@
+kind: ConfigMap
```

Comment:
Should we warn the user when they try to use cilium with k8s 1.7 for instance? It will not work.